src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -265,6 +268,13 @@ enum class AspectRatioMeetOrSlice
     Slice
 };
 
+enum class GaussianBlurEdgeMode
```

Comment:
ThorVG APIs already provide the necessary functions and guidance. The SVG loader only needs such an `int gaussianBlurEdgeMode` to convert the SVG input to ThorVG.